common/src/main/resources/assets/estrogen/lang/de_de.json

```
@@ -18,50 +24,97 @@
   "attribute.name.estrogen.boob_growing_start_time": "Oberkörper-Startzeit",
   "attribute.name.estrogen.boob_initial_size": "Initiale Oberkörpergröße",
   "attribute.name.estrogen.dash_level": "Dash-Level",
+  "attribute.name.estrogen.fall_damage_resistance": "Fallschadenresistenz",
   "block.estrogen.centrifuge": "Zentrifuge",
   "block.estrogen.cookie_jar": "Keksdose",
   "block.estrogen.cookie_jar.tooltip": "Keksdose",
   "block.estrogen.cookie_jar.tooltip.summary": "Eine Dose zum _aufbewahren_ aller deiner _Kekse_!",
+  "block.estrogen.dormant_dream_block": "Schlafender Traumblock",
+  "block.estrogen.dormant_dream_block.tooltip": "Schlafender Traumblock",
+  "block.estrogen.dormant_dream_block.tooltip.behaviour1": "Während der _Nacht_ §kwird er aktiv",
+  "block.estrogen.dormant_dream_block.tooltip.condition1": "Wenn durch ein Redstone Signal aktiviert",
+  "block.estrogen.dormant_dream_block.tooltip.summary": "Ein Block mit einigen _sehr_ mächtigen Eigenschaften...",
   "block.estrogen.dream_block": "Flasche der Träume",
+  "block.estrogen.estrogen_pill_block": "Östrogen Pillen Schachtel",
   "block.estrogen.filtrated_horse_urine": "Filtriertes Pferdeurin",
+  "block.estrogen.gender_fluid": "Flüssiges Gender",
   "block.estrogen.horse_urine": "Pferdeurin",
   "block.estrogen.liquid_estrogen": "Flüssiges Östrogen",
   "block.estrogen.molten_amethyst": "Geschmolzener Amethyst",
   "block.estrogen.molten_slime": "Geschmolzener Schleim",
+  "block.estrogen.moth_bed": "Rosarotes Ahornbett",
+  "block.estrogen.moth_seat": "Rosaroter Ahornsitz",
+  "block.estrogen.moth_wool": "Rosarotes Ahornwolle",
+  "block.estrogen.moth_wool_carpet": "Rosaroter Ahornwollteppich",
+  "block.estrogen.quilted_moth_bed": "Gestepptes rosarotes Ahornbett",
+  "block.estrogen.quilted_moth_wool": "Gesteppte rosarote Ahornwolle",
+  "block.estrogen.quilted_moth_wool_carpet": "Gesteppter rosaroter Ahornwollteppich",
   "block.estrogen.testosterone_mixture": "Testosteron-Mixtur",
   "category.estrogen": "Östrogen",
   "create.recipe.centrifuging": "Zentrifugieren",
   "create.recipe.entity_interaction": "Tierinteraktion",
-  "curios.identifier.thighs": "Oberschenkel",
+  "curios.identifier.thighs": "Oberschenkelstrümpfe",
+  "curios.modifiers.thighs": "Oberschenkelstrümpfe Modifikatoren",
   "death.attack.girlpower": "%s hat zu viel Frauenpower verwendet",
   "death.attack.girlpower.player": "%s hat zu viel Frauenpower verwendet",
   "effect.estrogen.estrogen": "Frauenpower",
+  "effect.estrogen.estrogen.description": "Erlaubt den Spielern zu Dashen und gibt ihnen ein paar zusätliche \"features\"",
```

```suggestion
  "effect.estrogen.estrogen.description": "Erlaubt dem Spieler zu Dashen und gibt ihm ein paar zusätliche \"features\"",
```

The English version uses the singular. Not sure if you *intentionally* switched to plural here to avoid the generic masculine (which I think is fair, *especially* in the context of... \*checks notes\* taking estrogen to gain "girl power" and grow boobs), but I thought I'd at least mention it in case it *wasn't* on purpose.

---

That was on purpose :)